cartography/intel/kubernetes/pods.py

```diff
@@ -0,0 +1,83 @@
+import logging
+from typing import Dict
+from typing import List
+
+from neo4j import Session
+
+from cartography.intel.kubernetes.util import get_epoch
+from cartography.intel.kubernetes.util import K8sClient
+from cartography.util import timeit
+
+logger = logging.getLogger(__name__)
+
+
+@timeit
+def sync_pods(
+    session: Session, client: K8sClient, update_tag: int, cluster: Dict,
+) -> List[Dict]:
+    pods = list()
+    for pod in client.core.list_pod_for_all_namespaces().items:
+        pods.append(
+            {
+                "uid": pod.metadata.uid,
+                "name": pod.metadata.name,
+                "creation_timestamp": get_epoch(pod.metadata.creation_timestamp),
+                "deletion_timestamp": get_epoch(pod.metadata.deletion_timestamp),
+                "namespace": pod.metadata.namespace,
+                "node": pod.spec.node_name,
+                "cluster_uid": cluster["uid"],
+                "labels": pod.metadata.labels,
+                "containers": [
+                    {
+                        "name": container.name,
+                        "image": container.image,
+                        "uid": f"{pod.metadata.uid}-{container.name}",
+                    }
+                    for container in pod.spec.containers
+                ],
+            },
+        )
```

Comment:
Code structure issue - 
Put this in a `get_pods` function and call it in the sync.
Refer one of the AWS codes for what the standard code structure is.